PersonStageThree/src/com/beyond/person/helper/UserInput.java

```diff
@@ -0,0 +1,73 @@
+package com.beyond.person.helper;
+
+import com.beyond.person.core.exception.InvalidFieldException;
+import com.beyond.person.core.exception.InvalidTypeException;
+import com.beyond.person.core.model.BasePerson;
+import com.beyond.person.core.model.Dancer;
+import com.beyond.person.core.model.Programmer;
+import com.beyond.person.core.model.Singer;
+import com.beyond.person.core.service.factory.PersonFactory;
+import com.beyond.person.core.service.implementation.DancerImpl;
+import com.beyond.person.core.service.implementation.ProgrammerImpl;
+import com.beyond.person.core.service.implementation.SingerImpl;
+
+import java.util.Scanner;
+
+public class UserInput {
+
+    /**
+     * method for creating the person needed and preforming the actions needed
+     *
+     * @param choice
+     * @param name
+     * @param lastName
+     * @param designation
+     */
+    public void userInput(int choice, String name, String lastName, String designation) throws InvalidFieldException {
+        Scanner enter = new Scanner(System.in);
+        switch (choice) {
+            case 1:
+                System.out.println("What group does the dancer dance in");
```

Comment:
@maria-salmasian putt such big logic on switch case it's notable  good idea

Reply:
changed it to if else